FeatureTests/Features/AnswerScreenTests.swift

```diff
@@ -59,54 +59,6 @@ class AnswerScreenTests: XCTestCase {
             .assert(grey_verifyDeviceAgnosticSnapshot())
     }
 
-    func testPositiveButtonResponseAfterAlertDismiss() {
```

Comment:
Are you positive that these tests should be removed?

Reply:
Yes, I wrote them and they were used in specific case on one issue with non-responsive screen. There is no such case for these tests anymore.